nixos/modules/services/networking/wstunnel.nix

```diff
@@ -1,123 +1,152 @@
-{ config, lib, options, pkgs, utils, ... }:
-with lib;
+{ config
+, lib
+, pkgs
+, ...
+}:
+
 let
   cfg = config.services.wstunnel;
-  attrsToArgs = attrs: utils.escapeSystemdExecArgs (
-    mapAttrsToList
-    (name: value: if value == true then "--${name}" else "--${name}=${value}")
-    attrs
+
+  attrsToArgs = attrs: lib.escapeShellArgs (
+    lib.mapAttrsToList
+      (name: value: if value == true then "--${name}" else "--${name}=${value}")
+      attrs
   );
```

Comment:
If you decide to go for the `lib.cli.toGNUCommandLineShell` thing, this becomes obsolete.

```suggestion
```